ArduCopter/mode_guided.cpp

```diff
@@ -2,36 +2,6 @@
 
 #if MODE_GUIDED_ENABLED
 
-/*
- * Init and run calls for guided flight mode
- */
-
-static Vector3p guided_pos_target_cm;       // position target (used by posvel controller only)
```

Comment:
Lots of these things need to be static. for guided to work as part of auto for example.